### DISCUSSION TITLE
Can PUMGen convert .xdmf format file to .msh format?

### DISCUSSION THREAD

**Discussion by buzhiqiuyuebai** [SeisSol/SeisSol#717] [Q&A] [answered]
I can use PUMGen(https://github.com/SeisSol/PUMGen) convert `.msh` format file to `.xdmf` format, but I don't know can PUMGen convert .xdmf format file to .msh format?

**Comment by krenzland** [comment, marked as answer]
It cannot.

**Reply by buzhiqiuyuebai** [reply]
> It cannot.

OK,thank you.
